packages/fxa-auth-server/lib/senders/templates/_versions.json

```diff
@@ -1,32 +1,33 @@
 {
-  "lowRecoveryCodesEmail": 4,
-  "newDeviceLoginEmail": 4,
-  "passwordChangedEmail": 3,
-  "passwordResetEmail": 3,
-  "passwordResetRequiredEmail": 3,
-  "postChangePrimaryEmail": 3,
-  "postRemoveSecondaryEmail": 3,
-  "postVerifyEmail": 3,
-  "postVerifySecondaryEmail": 3,
-  "postVerifyTrailheadEmail": 3,
-  "recoveryEmail": 3,
+  "downloadSubscription": 1,
+  "lowRecoveryCodes": 5,
```

Comment:
The template names in metrics now match the template names in the codebase, which improves greppability and debuggability when investigating things that show up in metrics.

Of course, it comes at the cost of a one-time discontinuity in the data, but we've been through those bumps before and they quickly fade with time. I think the naming consistency is a win worth paying that cost for.

I also bumped the version numbers, although that wasn't strictly necessary.